CMakeLists.txt

```diff
@@ -315,6 +315,31 @@ if(LAPACKE)
   add_subdirectory(LAPACKE)
 endif()
 
+# --------------------------------------------------
+# BLAS++ / LAPACK++
+option(BLAS++ "Build BLAS++" OFF)
+option(LAPACK++ "Build LAPACK++" OFF)
+
+if (BLAS++)
+  message(STATUS "BLAS++ enable")
+  message(STATUS "----------------")
+  message(STATUS "Thank you for your interest in BLAS++, a newly developed C++ API for BLAS library")
+  message(STATUS "The objective of BLAS++ is to provide a convenient, performance oriented API for development in the C++ language, that, for the most part, preserves established conventions, while, at the same time, takes advantages of modern C++ features, such as: namespaces, templates, exceptions, etc.")
+  message(STATUS "We are still working on integrating BLAS++ in our library. For the moment, you can download directly blas++ from https://bitbucket.org/icl/blaspp")
+  message(STATUS "For support BLAS++ related question, please email: slate-user@icl.utk.edu")
+  message(STATUS "----------------")
+endif()
+
+if (LAPACK++ )
+  message(STATUS "LAPACK++ enable")
+  message(STATUS "----------------")
+  message(STATUS "Thank you for your interest in LAPACK++ , a newly developed C++ API for LAPACK library")
+  message(STATUS "The objective of LAPACK++ is to provide a convenient, performance oriented API for development in the C++ language, that, for the most part, preserves established conventions, while, at the same time, takes advantages of modern C++ features, such as: namespaces, templates, exceptions, etc.")
+  message(STATUS "We are still working on integrating LAPACK++ in our library. For the moment, you can download directly blas++ from https://bitbucket.org/icl/lapackpp")
+  message(STATUS "For support LAPACK++ related question, please email: slate-user@icl.utk.edu")
+  message(STATUS "----------------")
+endif()
```

Comment:
To ensure consistency in the message, I instead suggest something like this:


```
function(_display_cpp_implementation_msg name)
  string(TOLOWER ${name} name_lc)
  message(STATUS "${name}++ enable")
  message(STATUS "----------------")
  message(STATUS "Thank you for your interest in ${name}++, a newly developed C++ API for ${name} library")
  message(STATUS "The objective of ${name}++ is to provide a convenient, performance oriented API for development in the C++ language, that, for the most part, preserves established conventions, while, at the same time, takes advantages of modern C++ features, such as: namespaces, templates, exceptions, etc.")
  message(STATUS "We are still working on integrating ${name}++ in our library. For the moment, you can download directly ${name_lc}++ from https://bitbucket.org/icl/${name_lc}pp")
  message(STATUS "For support ${name}++ related question, please email: slate-user@icl.utk.edu")
  message(STATUS "----------------")
endif()
if(BLASPP)
  _display_cpp_implementation_msg("BLAS")
endif()
if(LAPACKPP)
  _display_cpp_implementation_msg("LAPACK")
endif()

```

Reply:
@mgates3  As you can see in the text above, the actual project is still describe as "${name}++" where `name` could be either `LAPACK` or `BLAS`